src/calcium/qqbar.jl

```diff
@@ -10,15 +10,15 @@
 #
 ###############################################################################
 
-parent(a::QQBarFieldElem) = CalciumQQBar
+parent(a::QQBarFieldElem) = QQBar
```

Comment:
Could just avoid the global var altogether, since it is a singleton it is equivalent to write
```suggestion
parent(a::QQBarFieldElem) = QQBarField()
```